test/tests/api-cit/v2_0/obm_tests.py

```diff
@@ -57,5 +57,5 @@ def test_setup_snmp_obm_api2(self):
     @depends(after='test_setup_snmp_obm_api2')
     def test_check_snmp_obm_settings(self):
         # """ Checking SNMP OBM settings GET:/nodes """
-        self.assertNotEqual(len(obmSettings().check_nodes(service_type='snmp-obm-service')), 0,
-                            msg='There are missing SNMP OBM settings!')
+        self.assertEqual(len(obmSettings().check_nodes(service_type='snmp-obm-service')), 0,
```

Comment:
Either the assert is wrong or the message is wrong? We assert that the number of nodes with snmp-obm-service must be 0 but then state that there SNMP OBM settings are missing. I.E., we will blow up when there ARE settings!